addons/sys_rack/stringtable.xml

```diff
@@ -98,7 +98,7 @@
             <Italian>Cruscotto</Italian>
         </Key>
         <Key ID="STR_ACRE_sys_rack_dashLower">
-            <English>Dashboard Lower</English>
+            <English>Dash Lower</English>
```

Comment:
Decided it will be changed if that's true. AFAIK "Dash" can just be used as short form of "Dashboard", and it fits interaction menu in a much nicer way.

Reply:
Dash is short for dashboard.

"Don't put you coffee on the dash."

Reply:
Perfect then. My bad.